### PR TITLE
Test Written for compound assignment operator (+=,-=)

### PR DESCRIPTION
**Pull request by hiteshbhavsar**
Hello @RasmusBroborg, I have made written the test required for the compound assignment operator += and -=. I request you review them and let me know if there are any changes that are required.

Best Regards.
Hitesh

### CODE REVIEW

File: tests/timecode.test/Timecode_Test.cs [hunk 1]
```diff
@@ -235,6 +235,38 @@ public void Timecode_Add_AddsTimecode()
...
+      //Arrange
+      var t1 = new Timecode(11, 11, 11, 0, Enums.Framerate.fps29_97_NDF);
+      var t2 = new Timecode(1, 1, 1, 0, Enums.Framerate.fps29_97_NDF);
+
+      //Act 
+      t1 -= t2;
+      //Assert
+      Assert.Equal(10, t1.Hour);
+      Assert.Equal(10, t1.Minute);
+      Assert.Equal(10, t2.Second);
```
Comment (RasmusBroborg) [inline tests/timecode.test/Timecode_Test.cs:250]: This should be t1.Second

File: tests/timecode.test/Timecode_Test.cs [hunk 2]
```diff
@@ -235,6 +235,38 @@ public void Timecode_Add_AddsTimecode()
...
+      //Arrange
+      var t1 = new Timecode(11, 11, 11, 0, Enums.Framerate.fps29_97_NDF);
+      var t2 = new Timecode(1, 1, 1, 0, Enums.Framerate.fps29_97_NDF);
+
+      //Act 
+      t1 += t2;
+      //Assert
+      Assert.Equal(12, t1.Hour);
+      Assert.Equal(12, t1.Minute);
+      Assert.Equal(12, t2.Second);
```
Comment (RasmusBroborg) [inline tests/timecode.test/Timecode_Test.cs:266]: This should be t1.Second

**Review by RasmusBroborg — CHANGES_REQUESTED** [review]
@hiteshbhavsar  Thank you for submitting test cases which verify the integrity of the Timecode class. 

I have spotted two lines which need to be revised before the Pull Request can be accepted.

Best,
Rasmus

File: tests/timecode.test/Timecode_Test.cs (hunk 1 above)
Comment (hiteshbhavsar) [reply tests/timecode.test/Timecode_Test.cs:250]: Thank you for letting me know that.

File: tests/timecode.test/Timecode_Test.cs (hunk 2 above)
Comment (hiteshbhavsar) [reply tests/timecode.test/Timecode_Test.cs:266]: Thank you for letting me know that.

**Review by RasmusBroborg — APPROVED** [review]
Hi @hiteshbhavsar,
Your additions look great and all tests have passed their checks, confirming your theory. I will approve the PR and merge with main. 

This PR closes #47.

**Pull request merged**